src/tests/posix/pthread/pthread_kill_test.c

```diff
@@ -0,0 +1,109 @@
+/**
+ * @file
+ * @brief test for pthread_kill
+ *
+ * @author  XU Chenglin
+ * @date    09/03/2020
+ */
+
+#include <pthread.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+#include <signal.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("posix/pthread_kill");
+
+void sighand(int signo) {
+    test_assert_not_zero(signo);
```

Comment:
I mean than you can check that right signal handler has been called. Using different handlers for example or set some variables

Reply:
I emit different characters in two different handlers. I set them as handler for different Signals. I send only one Signal and capture it after join() with test_assert_get_emitted(). So I can see if the handler called is good.